resolwe_bio/processes/slamdunk/alleyoop_rates.py

```diff
@@ -0,0 +1,64 @@
+"""Run Alleyoop rates tool on Slamdunk results."""
+import os
+
+from plumbum import TEE
+
+from resolwe.process import Cmd, DataField, FileField, Process, StringField
+
+
+class AlleyoopRates(Process):
+    """Run Alleyoop rates."""
+
+    slug = 'alleyoop-rates'
+    process_type = 'data:alleyoop'
+    name = 'Alleyoop Rates'
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image': 'resolwebio/slamdunk:1.0.0'
+            },
+        },
+        'resources': {
+            'cores': 1,
+            'memory': 16384,
+        },
+    }
+    entity = {
+        'type': 'sample',
+    }
+    data_name = '{{ reads|sample_name|default("?") }}'
+    version = '1.0.0'
+
+    class Input:
+        """Input fields for AlleyoopRates."""
+
+        ref_seq = DataField('seq:nucleotide', label='FASTA file containig sequences for alingnig')
+        slamdunk = DataField('alignment:bam:slamdunk', label='Slamdunk results')
+        
+    class Output:
+        """Output fields to process AlleyoopRates."""
+
+        rates = FileField(label='CSV file containing the overall conversion rates')
```

Comment:
The result looks like a TSV to me. Maybe make it more like the label for `plot` field, i.e. "Sequences for aligning". Note that you have a typo in "alingnig".

Reply:
Currently I have the maximum line length set to 119, however, most of the lines are under 100 with the exception of `return_code, _, _ = Cmd['alleyoop']['rates'][args][inputs.slamdunk.bam.path] & TEE(retcode=None)`. Should I aim to fit them into 80 or 100 characters?

Reply:
Looks like PEP is not adamant about line widths (except for docstrings) and our codebase seems to be more than 80. I think it's fine as it is.